From d65ef1d0ddf9a0e7e7d6fb9d4c9d3319d2b311f1 Mon Sep 17 00:00:00 2001
From: aaaa <aaaa@qq.com>
Date: Sun, 28 May 2023 13:44:07 +0900
Subject: [PATCH] fix build

---
src/nft_ext_fullcone.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/nft_ext_fullcone.c b/src/nft_ext_fullcone.c
index ef36b56..55f0c76 100644
--- a/src/nft_ext_fullcone.c
+++ b/src/nft_ext_fullcone.c
@@ -124,7 +124,7 @@
-static int nft_fullcone_validate(const struct nft_ctx *ctx, const struct nft_expr *expr, const struct nft_data **data)
+static int nft_fullcone_validate(const struct nft_ctx *ctx, const struct nft_expr *expr)
 {
        int err;

        err = nft_chain_validate_dependency(ctx->chain, NFT_CHAIN_T_NAT);

--
